config/.env.sample

```diff
@@ -47,6 +47,8 @@ CANVAS_INSTANCE_URL=https://canvas-test.it.umich.edu
 CANVAS_API_CLIENT_ID=
 # Secret for the Canvas Developer Key associated with the application
 CANVAS_API_SECRET=
+# The canvas custom roles id's that used when enrolling users to section, this has a fallback value if not set
+CANVAS_CUSTOM_ROLES={"Assistant": 1, "Librarian": 2}
```

Comment:
This starts to address the problems I identified in #339, but it's incomplete.  That is, it identifies some enrollment types, but it doesn't specify who can assign these types.